tests/communities/test_communities.py

```diff
@@ -142,7 +130,6 @@ def test_edit_community(main_screen: MainWindow, params):
         assert community_info.name == params['name']
         assert community_info.description == params['description']
         assert '1' in community_info.members
-        image.compare(community_info.image, 'logo_in_settings_updated.png')
```

Comment:
Let's skip these steps without deleting them, I will fix it later.

Reply:
Ok. I commented these steps and left link to a separate task - to replace in these steps and also https://github.com/status-im/desktop-qa-automation/blob/master/gui/screens/community.py#L44 - here. I will assign it to you then

Reply:
i suggest to wait a bit with this task. I dont see any strong pros for testing through comparing images right now, there is other stuff to cover. @StateOf-Vlado we still can have the task and the code, but implementation should be deferred for now.